extensions/typescript-language-features/package.nls.json

```
@@ -124,6 +124,10 @@
 	"typescript.preferences.importModuleSpecifierEnding.minimal": "Shorten `./component/index.js` to `./component`.",
 	"typescript.preferences.importModuleSpecifierEnding.index": "Shorten `./component/index.js` to `./component/index`.",
 	"typescript.preferences.importModuleSpecifierEnding.js": "Do not shorten path endings; include the `.js` extension.",
+	"typescript.preferences.jsxAttributeCompletionStyle": "Preferred style for JSX attributes completions.",
+	"typescript.preferences.jsxAttributeCompletionStyle.auto": "Automatically select the completion style based on the attribute type.",
+	"typescript.preferences.jsxAttributeCompletionStyle.braces": "Enclosed in braces all of the JSX attribute values.",
+	"typescript.preferences.jsxAttributeCompletionStyle.none": "Don't apply any completions on JSX attribute values.",
```

What do you think about
```suggestion
	"typescript.preferences.jsxAttributeCompletionStyle.auto": "Insert `={}` or `=""` after attribute names based on the prop type.",	
	"typescript.preferences.jsxAttributeCompletionStyle.braces": "Insert `={}` after attribute names.",
	"typescript.preferences.jsxAttributeCompletionStyle.none": "Only insert attribute names.",
```

(The backticks will get interpreted and markdown and show up formatted as code.)

---

Sounds good. I was actually struggling to get to a good description.